docs/examples/multi-agent-workflow.md

```diff
@@ -0,0 +1,233 @@
+In this example we implement a multi-agent Workflow we have a single "Researcher" Agent that conducts research on a topic, and then fans out to multiple "Writer" Agents to write blog posts based on the research.
+
+By splitting up our workloads across multiple Structures, we can parallelize the work and leverage the strengths of each Agent. The Researcher can focus on gathering data and insights, while the Writers can focus on crafting engaging narratives.
+Additionally, this opens us up to the possibility of having each Agent run on a separate machine or container, allowing us to scale our Workflow as needed. 
+
+
+```python
+import json
+import os
+from pathlib import Path
+
+from griptape.drivers import WebhookEventListenerDriver
+from griptape.events import EventListener, FinishStructureRunEvent
+from griptape.rules import Rule, Ruleset
+from griptape.structures import Agent, Workflow
+from griptape.tasks import NoOpTask, StructureRunTask
+from griptape.tools import (
+    TaskMemoryClient,
+    WebScraper,
+    WebSearch,
+)
+
+
+def build_researcher():
```

Comment:
I know this is in a markdown file and meant to be consumed top-to-bottom, but is there a way to write this in a way where it's easier to transplant directly into Structures? (without it getting unwieldly)

Reply:
I'd like to keep docs easily copyable and runnable. When we have a course on building multi-agent workflows in Cloud we can link to that.

Reply:
I YIELD